src/app/components/client/toolbar/AppPicker.module.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../../tokens";
+
+.wrapper {
+  align-items: center;
+  position: relative;
+}
+
+.trigger {
+  display: flex;
+  cursor: pointer;
+  background-color: transparent;
+  border-style: none;
+  // Diameter of the Bento image:
+  width: 24px;
+  height: 24px;
```

Comment:
Question: Should we start using CSS variables for sizes?

Reply:
In this case, I'm inclined to say no, since this is the only place this specific value is (and should be) used, and I'd be really surprised if someone accidentally changes one but not the other. Though I won't die on this hill if you will :)

Reply:
No, all good.